lib/utils/findMediaFeatureNames.js

```diff
@@ -0,0 +1,144 @@
+'use strict';
+
+const { TokenType, isToken, stringify, tokenize } = require('@csstools/css-tokenizer');
+const {
+	isTokenNode,
+	parseCommaSeparatedListOfComponentValues,
+	isSimpleBlockNode,
+} = require('@csstools/css-parser-algorithms');
+const {
+	isGeneralEnclosed,
+	isMediaFeature,
+	isMediaQueryInvalid,
+	parseFromTokens,
+} = require('@csstools/media-query-list-parser');
```

Comment:
Just idea for future and future discussions, maybe we can union and write full featured CSS parser + at-rules/values parser from scratch, I am afraid we can't rewrite postcss due some specific logic (and it will probably take longer), so union around CSS parser will be great for any JS tooling, we can open an issue for this

Reply:
Yeah, @romainmenke What do you think about it?

Reply:
Sounds good!
But indeed best to discus in a separate issue,
I have too many thoughts on this subject :D